### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by ibrohimkhan**
Solution for homework week 2.

### CODE REVIEW

File: app/src/main/java/com/kodeco/android/coordplot/components/Map.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.R
+import com.kodeco.android.coordplot.ui.theme.AppTheme
+import com.kodeco.android.coordplot.ui.theme.primaryContainerDark
+
+@Composable
+fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
+    Box(
+        modifier = Modifier
+            .size(300.dp)
```
Comment (zlippard) [inline app/src/main/java/com/kodeco/android/coordplot/components/Map.kt:21]: You can also move these `dp` values and other constants up and out of the function to `private const val`s or `private val`s.

File: app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
     <style name="Theme.MyApplication" parent="android:Theme.Material.Light.NoActionBar" />
+
+    <style name="Theme.SplashScreen.MySplash" parent="Theme.SplashScreen">
```
Comment (zlippard) [inline app/src/main/res/values/themes.xml:5]: Great job adding this splash screen theme!

**Review by zlippard — APPROVED** [review]
Overall great job this week Ibrokhim! This project meets and exceeds the expectations for the assignment.  Only left one comment around constants, but overall it's minor. Keep up the great work! :]

File: app/src/main/java/com/kodeco/android/coordplot/components/Map.kt (hunk 1 above)
Comment (ibrohimkhan) [reply app/src/main/java/com/kodeco/android/coordplot/components/Map.kt:21]: hey @zlippard thanks for your feedback!

**Pull request merged**